lambdas/tests/unit/utils/test_get_aws_region.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from utils.get_aws_region import get_aws_region
+
+
+def test_get_aws_region_return_aws_region_in_env_var(mocker):
```

Comment:
these environment variables and test variables have been set as a fixture in `tests/unit/conftest.py` we can re use these

Reply:
Done